shelley/chain-and-ledger/formal-spec/epoch.tex

```diff
@@ -132,10 +132,12 @@ \subsection{Helper Functions and Accounting Fields}
 \begin{figure}[htb]
   \emph{Total possible refunds}
   \begin{align*}
-    & \fun{obligation} \in \PParams \to \StakeCreds \to \StakePools \to \Coin \\
-    & \obligation{pp}{stkCreds}{stpools}{cslot} =
-      (\fun{keyDeposit}~\var{pp}) \cdot|\var{stkCreds}| +
-      (\fun{poolDecayRate}~\var{pp}) \cdot|\var{stpools}| \\
+    & \fun{obligation} \in \PParams \to (\StakeCredential \mapsto \Coin)
+    \to (\KeyHash_{pool}\mapsto\PoolParam) \to \Coin \\
+    & \obligation{pp}{rewards}{poolParams} = \\
+    & ~~~~~
+    (\fun{keyDeposit}~\var{pp}) \cdot|\var{rewards}| +
+    (\fun{poolDecayRate}~\var{pp}) \cdot|\var{poolParams}| \\
```

Comment:
should we also remove the pool decay here?

Reply:
yes! thanks, good catch!